server/server.go

```diff
@@ -73,6 +76,7 @@ func Start(ctx context.Context, httpPort, httpsPort int, scaledContext *config.S
 	root.PathPrefix("/k8s/clusters/").Handler(authedHandler)
 	root.PathPrefix("/meta").Handler(authedHandler)
 	root.NotFoundHandler = ui.UI(http.NotFoundHandler())
+	root.PathPrefix("/v1-saml/").Handler(samlRoot)
```

Comment:
most of the other PathPrefix functions do not have a trailing /. I think yours should not have it either. The endpoint "/v1-saml" is technically your httpHandler's responsibility to handle (even if it just 404s).